test/support/macro/connection_helpers.ex

```diff
@@ -1,11 +1,12 @@
-defmodule Janus.HandlerTest.CallbackHelper do
-  # helper module that declares macro for testing `Janus.Handler`'s callback calls
-
-  # simple transport returning message given to `handle_info/2` without changes
-  # used by `Janus.Connection.handle_info/2`
-  defmodule SimpleTransport do
-    def handle_info(message, state) do
-      {:ok, message, state}
+defmodule Janus.Support.Macro do
+  defmacro assert_next_receive(pattern, timeout \\ 20000) do
```

Comment:
I don't quite grasp the idea of this function. What exactly does this do? Why not wrap around assert receive if that is really what you need/want?

Reply:
https://stackoverflow.com/questions/41543466/how-to-assert-elixir-messages-arrive-in-order